src/get-phantom-html.js

```diff
@@ -6,9 +6,13 @@ const Promise = require('bluebird');
 
 const scriptPath = path.join(__dirname, 'script.js');
 
-module.exports = (url) => {
+module.exports = (url, proxy) => {
+	console.log(proxy.hostname);
+	var host = proxy.hostname;
+	var port = proxy.port;
```

Comment:
I think we need to check that `proxy` exists on line 11 and 12

and remove console.log